### PR TITLE
Admin view pr changes

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: client/src/components/Employee/Employee.js [hunk 1]
```diff
@@ -11,13 +11,19 @@ import CustomInput from '../../components/CustomInput/CustomInput.js'
 import Select from '@material-ui/core/Select'
 import DateFnsUtils from '@date-io/date-fns'
 import InputLabel from '@material-ui/core/InputLabel'
-import { withToastManager, useToasts } from 'react-toast-notifications'
+import { useToasts } from 'react-toast-notifications'
 import Grid from '@material-ui/core/Grid'
-import checkboxAdnRadioStyle from '../../assets/jss/material-dashboard-react/checkboxAdnRadioStyle.js'
+import { employeeStyles } from '../../componentStyles/index'
```
Comment (bhushan-oe) [inline client/src/components/Employee/Employee.js]: by default it is index

File: client/src/components/Employee/EmployeeConstants.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import * as Yup from 'yup'
+
+export const formikInitialValues = userToUpdate => {
```
Comment (bhushan-oe) [inline client/src/components/Employee/EmployeeConstants.js]: use spread operator. e.g. 
`const {
  firstname, 
    lastname,
    middlename,
    address1,
    address2,
    city    
} = userToUpdate;`

File: client/src/components/Employee/EmployeeConstants.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  userName: Yup.string()
+    .min(8, 'UserName must be at least 8 characters long!')
+    .required('UserName is required'),
+  password: Yup.string()
+    .min(8, 'Password must be at least 8 characters long!')
+    .required('Password is required')
+})
+
+export const formikUpdateValidations = Yup.object().shape({
+  firstname: Yup.string()
```
Comment (bhushan-oe) [inline client/src/components/Employee/EmployeeConstants.js]: reduce duplicate code

File: client/src/components/Employee/EmployeeListing.js [hunk 4]
```diff
@@ -28,12 +32,8 @@ const EmployeeListing = props => {
   const [userToUpdate, setUserToUpdate] = useState()
   const [updateAction, setUpdateAction] = useState()
   const [showDelDialog, setShowDelDialog] = useState(false)
-  const deleteEmployeeSuccess = useSelector(
-    state => state.EmployeeInfo.deleteEmployeeSuccess
-  )
-  const deleteEmployeeError = useSelector(
-    state => state.EmployeeInfo.deleteEmployeeError
-  )
+  const deleteEmployeeSuccess = useSelector(state => deleteEmployeeSuccessMsg(state))
```
Comment (bhushan-oe) [inline client/src/components/Employee/EmployeeListing.js]: Can you try using  `useSelector(deleteEmployeeSuccessMsg)` ?

File: client/src/selectors/index.js [hunk 5]
```diff
@@ -1,5 +1,30 @@
 import setBrowserInfo from '../reducers/employeeInfo'
```
Comment (bhushan-oe) [inline client/src/selectors/index.js]: setBrowserInfo is not in use

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: client/src/components/Employee/EmployeeListing.js (hunk 4 above)
Comment (surekhag) [reply client/src/components/Employee/EmployeeListing.js]: Done

File: client/src/selectors/index.js (hunk 5 above)
Comment (surekhag) [reply client/src/selectors/index.js]: Removed

File: client/src/components/Employee/Employee.js (hunk 1 above)
Comment (surekhag) [reply client/src/components/Employee/Employee.js]: Updated structure for styles

File: client/src/components/Employee/EmployeeConstants.js (hunk 2 above)
Comment (surekhag) [reply client/src/components/Employee/EmployeeConstants.js]: Done

File: client/src/components/Employee/EmployeeConstants.js (hunk 3 above)
Comment (surekhag) [reply client/src/components/Employee/EmployeeConstants.js]: Done

**Review by bhushan-oe — APPROVED** [review]

**Pull request merged**